src/installer/pkg/archives/dotnet-nethost.proj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
+    <CreateSymbolsArchive Condition="'$(DotNetBuildFromSource)' != 'true'">true</CreateSymbolsArchive>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Archives" Version="$(MicrosoftDotNetBuildTasksArchivesVersion)" />
```

Comment:
Curious, what's the entry point in that package? Who is responsible for invoking it? I don't see any of the targets below call into it (depend on a target in it).